### PR TITLE
docs: remove fields filter warning

### PR DESCRIPTION
**Pull request by jannyHou**
Update docs after fixing #4992 

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: docs/site/Fields-filter.md [hunk 1]
```diff
@@ -27,9 +27,6 @@ query will include **only** those you specifically include with filters.
 
 ### REST API
 
-{% include warning.html content="
-As a known bug, the option `<false>` does not work for url for now. Please include the properties you need or use the [stringified JSON format](Querying-data.html#using-stringified-json-in-rest-queries) to meet your requirement. The bug is tracked in the GH issue [#4992](https://github.com/strongloop/loopback-next/issues/4992)" %}
-
 <pre>
 filter[fields][<i>propertyName</i>]=true&filter[fields][<i>propertyName</i>]=true...
```
[the remark this answers is not in the document]
Comment (jannyHou) [reply docs/site/Fields-filter.md]: updated 👍

**Comment by jannyHou** [comment]
@agnes512 Thank you, I adjusted the content a bit, now it's:

Return customer information only with `name` and `address`, and hide their `id`:

```ts
await customerRepository.find({fields: {id: false, name: true, address: true}});
```

Or use stringified JSON format:
`/orders?filter={"fields":{"name":true,"address":true,"id":false}}`

Include all required properties:
`/customers?filter[fields][name]=true&filter[fields][address]=true`

Or omit the hidden property:
`/customers?filter[fields][id]=false`

File: docs/site/Fields-filter.md [hunk 2]
```diff
@@ -50,14 +47,18 @@ Return customer information only with `name` and `address`, and hide their `id`:
 await customerRepository.find({fields: {id: false, name: true, address: true}});
 ```
 
+Or use stringified JSON format:
```
Comment (agnes512) [inline docs/site/Fields-filter.md:50]: Nitpick:
Other filters have the stringified JSON format example after the REST examples.

File: docs/site/Fields-filter.md [hunk 3]
```diff
@@ -50,14 +47,18 @@ Return customer information only with `name` and `address`, and hide their `id`:
...
 
-Include all required properties as a workaround:
+Include all required properties:
 `/customers?filter[fields][name]=true&filter[fields][address]=true`
 
-Or use stringified JSON format:
+Or omit the hidden property:
 
-`/orders?filter={"fields":{"name":true,"address":true,"id":false}}`
+`/customers?filter[fields][id]=false`
```
Comment (agnes512) [inline docs/site/Fields-filter.md:61]: Btw this might not be correct if the model has properties other than `id`, `name`, and `address`.

**Comment by jannyHou** [comment]
@agnes512 I just realized there are 2 separate examples for include and exclude, re-wrote the content, ptal thanks!

**Pull request merged**